src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/feedbackSubmissionForm.tag

```diff
@@ -47,7 +47,7 @@
                     There are no questions for you to answer here!
             </c:when>
             <c:otherwise>
-                <input type="checkbox" name="sendsubmissionemail" checked>
+                <input type="checkbox" name="sendsubmissionemail" checked value="true">
```

Comment:
Used this so that we can use `getRequestParamAsBoolean` in java file directly. Should I remove `value=true` and use approach as stated by you below ? Are there some cons of using the `value=true` approach ?